### PR TITLE
Do not override high_availability attribute

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

File: lib/fog/ovirt/compute/v4.rb [hunk 1]
```diff
@@ -82,6 +82,7 @@ def get_attr_value(value, opts)
               value
             when OvirtSDK4::HighAvailability
               opts[:ha] = value.enabled
+                value
```
Comment (adamruzicka) [inline lib/fog/ovirt/compute/v4.rb]: The indentation looks off.

**Comment by orrabin** [comment]
@adamruzicka yeah something was off in the indentation, I fixed it now, thanks!

**Review by adamruzicka — COMMENTED** [review]
Not much to add here, shall we wait until the rubocop issues are solved in another PR?

**Comment by orrabin** [comment]
already open: https://github.com/fog/fog-ovirt/pull/40
I would suggest merging that one first and rebasing this

**Comment by orrabin** [comment]
@adamruzicka I rebased with the rubocop fixes

**Comment by adamruzicka** [comment]
It looks like we need some additional rubocop fixes, opened https://github.com/fog/fog-ovirt/pull/43 to address that

**Comment by adamruzicka** [comment]
One more rebase please :)

**Comment by orrabin** [comment]
@adamruzicka tests are finally green!

**Review by adamruzicka — APPROVED** [review]
ACK

**Comment by adamruzicka** [comment]
Thanks @orrabin !

**Pull request merged**